scan/lib/scan/options.rb

```diff
@@ -537,7 +537,12 @@ def self.available_options
                                     env_name: "SCAN_FAIL_BUILD",
                                     description: "Should this step stop the build if the tests fail? Set this to false if you're using trainer",
                                     type: Boolean,
-                                    default_value: true)
+                                    default_value: true),
+        FastlaneCore::ConfigItem.new(key: :package_authorization_provider,
```

Comment:
Could you add a `verify_block` to config items (in scan, snapshot and gym) to check that the value is either `keychain` or `netrc` in a similar way as it's done for `catalyst_platform` above?

Reply:
thanks for the feedback! Done and tested to verify it works